workflow-bot-app/src/main/java/com/symphony/bdk/workflow/engine/executor/message/PinMessageExecutor.java

```diff
@@ -0,0 +1,45 @@
+package com.symphony.bdk.workflow.engine.executor.message;
+
+import com.symphony.bdk.core.util.IdUtil;
+import com.symphony.bdk.gen.api.model.V1IMAttributes;
+import com.symphony.bdk.gen.api.model.V3RoomAttributes;
+import com.symphony.bdk.gen.api.model.V4Message;
+import com.symphony.bdk.workflow.engine.executor.ActivityExecutor;
+import com.symphony.bdk.workflow.engine.executor.ActivityExecutorContext;
+import com.symphony.bdk.workflow.swadl.v1.activity.message.PinMessage;
+
+import lombok.extern.slf4j.Slf4j;
+
+import java.io.IOException;
+
+@Slf4j
+public class PinMessageExecutor implements ActivityExecutor<PinMessage> {
+
+  public static final String IM = "IM";
+  public static final String ROOM = "ROOM";
+
+  @Override
+  public void execute(ActivityExecutorContext<PinMessage> execution) throws IOException {
+    String messageId = IdUtil.toUrlSafeIdIfNeeded(execution.getActivity().getMessageId());
```

Comment:
let's maybe comment this a temporary workaround?

Reply:
sure done in the next commit